src/main/python/ui/postbuilder.py

```diff
@@ -223,27 +291,26 @@ def setupUi(self, postbuilder):
         postbuilder.setTabOrder(self.ddCheckBox, self.lpcm71CheckBox)
         postbuilder.setTabOrder(self.lpcm71CheckBox, self.lpcm51CheckBox)
         postbuilder.setTabOrder(self.lpcm51CheckBox, self.postTextEdit)
-        postbuilder.setTabOrder(self.postTextEdit, self.generateButton)
-        postbuilder.setTabOrder(self.generateButton, self.closeButton)
+        postbuilder.setTabOrder(self.postTextEdit, self.fileName)
+        postbuilder.setTabOrder(self.fileName, self.tmdbButton)
+        postbuilder.setTabOrder(self.tmdbButton, self.generateButton)
+        postbuilder.setTabOrder(self.generateButton, self.audioValid)
```

Comment:
a few layout comments

* the "valid" buttons look like buttons rather than statuses, partly because they are clickable & they are also blank before you enter any text
* would be nice to put the tmdb button next to the tmdb fields (e.g. after the tmdb id button)
* it's not obvious how to use the tmdb button, particularly for search vs loading an id
* some feedback (like a spinner) while it is searc
* suggest adding logging to indicate which URL it just hit and what the results were, gives some way to see what is going on (even if there is no UI for search results)

Reply:
- For the "valid" buttons I'm assuming you mean to not have them in the tab order at all.  The UI editor did that when I fixed the tab order for the fields.  I'll have to figure out how to remove them. On my mac I can't tab past the post text field and assumed it was like that on other platforms so I didn't worry about the items after it.  

- moving the button is easy

- any suggestion how to clarify that it searches and loads if no ID but only loads if there is an id?  I didn't really want to use two buttons for it.  Overloading seemed easy enough to "get" since it's mainly going to be Mo using it.

- Gah, now I have to learn that in QT ;) 

- easy enough to add.